images/linux/scripts/installers/preparemetadata.sh

```diff
@@ -9,8 +9,18 @@ source $HELPER_SCRIPTS/document.sh
 
 WriteItem "<!--- DO NOT EDIT - This markdown file is autogenerated. -->"
 
-Announcements=$(cat $HELPER_SCRIPTS/announcements.md)
-WriteItem $Announcements
+if [ $ANNOUNCEMENTS ]; then
+    Announcements="| Announcements |\n|-|\n"
+    IFS=';;' read -ra rawAnnouncements <<< "$UBUNTU_ANNOUNCEMENTS"
+    for rawAnnouncement in "${rawAnnouncements[@]}"; do
+        if [ $rawAnnouncement ]; then
+            IFS="===" read -ra announcement <<< "$rawAnnouncement"
+            Announcements+="| [${announcement[0]}](${announcement[3]}) |\n"
+        fi
+    done
+    WriteItem $Announcements
+fi
```

Comment:
Why can't we pass ready table to the packer template? 
Why we need to generate it? I think it will cause a lot of duplicated logic between macos, linux, windows

Reply:
Done, thank you!